pom.xml

```diff
@@ -109,7 +109,10 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>2.9</version>
+                <version>3.0.1</version>
```

Comment:
Was this change intentional?

Reply:
yes, but we can remove the javadoc plugin if we do not want to produce javadoc.